rest/route/annotations_test.go

```diff
@@ -465,7 +475,7 @@ func TestAnnotationByTaskPutHandlerRun(t *testing.T) {
 	assert.Equal(t, "task-1-note_0", annotation.Note.Message)
 	assert.Equal(t, "test_annotation_user", annotation.Note.Source.Author)
 	assert.Equal(t, "api", annotation.Note.Source.Requester)
-
+	assert.Equal(t, "api", annotation.Issues[0].Source.Requester)
```

Comment:
Can you add something to the "test update" below to verify that if Issues/SuspectedIssues isn't defined then it won't be updated?

Reply:
I don't need to see it again for you to merge though.